frontend/src/Components/StudentMetrics.tsx

```diff
@@ -0,0 +1,197 @@
+import { useEffect, useState } from 'react';
+import useSWR from 'swr';
+import { AxiosError } from 'axios';
+import API from '@/api/api';
+import { Facility, LoginMetrics, ServerResponseOne } from '@/common';
+import {
+    BarChart,
+    CartesianGrid,
+    XAxis,
+    YAxis,
+    Tooltip,
+    Legend,
+    Bar
+} from 'recharts';
+import PrimaryButton from './PrimaryButton';
+
+const OperationalInsights = () => {
```

Comment:
if the page is called student metrics it should probably align with the exported component in this page to clarify confusion